plexmuxy/config.py

```
@@ -0,0 +1,379 @@
+    if value in {"none", "move", "delete"}:
+        return value
+    raise ConfigError(f'{field_name} must be one of "none", "move", or "delete"')
+
+
+def name_strategy(value: Any, field_name: str) -> str:
+    if value in {"suffix", "same-name", "template"}:
+        return value
+    raise ConfigError(f'{field_name} must be one of "suffix", "same-name", or "template"')
+
+
+def optional_path(value: Any) -> Path | None:
```

Ruff (0.15.18)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)